test/e2e/app/app.go

```diff
@@ -296,6 +334,9 @@ func (app *Application) InitChain(_ context.Context, req *abci.InitChainRequest)
 	}
 
 	params := app.updateFeatureEnableHeights(0)
+	if params, err = app.flipConsensusParams(params, 0); err != nil {
```

Comment:
Why the height is 0 instead of `req.InitialHeight`? There are many uses of height 0 in `InitChain` so I guess there must be a reason (add a comment?).

Reply:
This is a good question. But in this case we have to fix the previous line too.

Do you mind opening an issue? So we can merge this standing PR.